README.md

```diff
@@ -69,7 +69,25 @@ dependencies {
 
 ### Usage
 
-// TODO
+Create new `Search Manager`
+```java
+var searchManager = new SearchManager();
```

Comment:
I think you should still mention the need to register sources here
just not with a code example including the creating of a source